invokeai/frontend/web/src/features/modelManagerV2/hooks/useIsImported.ts

```diff
@@ -0,0 +1,54 @@
+import { useCallback } from "react";
+import { ALL_BASE_MODELS } from "../../../services/api/constants";
+import { useGetMainModelsQuery, useGetLoRAModelsQuery, useGetTextualInversionModelsQuery, useGetControlNetModelsQuery, useGetT2IAdapterModelsQuery, useGetIPAdapterModelsQuery, useGetVaeModelsQuery, } from "../../../services/api/endpoints/models";
+import { EntityState } from "@reduxjs/toolkit";
+import { forEach } from "lodash-es";
+import { AnyModelConfig } from "../../../services/api/types";
+
+export const useIsImported = () => {
```

Comment:
@psychedelicious I feel like you will have a much better way to do this but I had a hard time figuring out how to compare the name from scanned models across model type queries